test/unit/bridge-client.test.js

```diff
@@ -68,11 +70,11 @@ describe('bridge-client', function () {
                 });
             `, context);
 
-            expect(context.__uvm_dispatch).be.a('function');
+            expect(context.__uvm_dispatch).to.be.a('function');
             context.__uvm_dispatch(JSON.stringify(['loopback', 'event-arg']));
```

Comment:
This should be changed to an assertion to check for specific invocation behaviour.